index.html

```diff
@@ -998,16 +998,90 @@
 
 </div>
 
-## Safety Systems {#safety}
+## Safety, Abuse, and Fraud {#safety}
+
+There are some cases where privacy and safety are in tension. These cases are more rare
+than one might think at first glance, but they still exist. We can distinguish three
+categories:
```

Comment:
A person's privacy also rests on implicit safety trade-offs: In order to protect accounts from getting hijacked, services may fingerprint a user's machine and require additional challenges when the account is accessed from a new device. Without stronger protections, retiring this fingerprinting would lead to more account compromises, and the compromise of all private information stored in these accounts.

Would it be possible to accommodate this (or another) example of privacy-enhancing mechanisms themselves requiring "appropriate use of data?"

Reply:
https://pr-preview.s3.amazonaws.com/jyasskin/privacy-principles/pull/105.html#example-account-security now mentions account hijacking.